Readme.md

```diff
@@ -23,3 +23,13 @@ __GIT utilities__ -- repo summary, repl, changelog population, author commit per
 Interested in contributing? Awesome!
 
 Please read [Contributing](CONTRIBUTING.md) before you make a PR, thanks!
+
+## Main branch
+As of Git Extras 6.4 the assumed default branch name changed from `master` to `main`.
```

Comment:
Need a blank line after the title, and we should only use one blank line between phases.